Router.d.ts

```diff
@@ -1,6 +1,6 @@
 ///<reference types="svelte" />
 
-import {SvelteComponent} from 'svelte'
+import {SvelteComponent, ComponentType} from 'svelte'
```

Comment:
Will this work with Svelte 3 too?

Reply:
The docstring for `ComponentType` looks identical, so I would assume so.

Reply:
Created test PRs on my repo that works on the example in this repo:

[Svelte 4 test](https://github.com/laat/svelte-spa-router/pull/2)
[Svelte 3 test](https://github.com/laat/svelte-spa-router/pull/1)

Reply:
Thanks for confirming!!